src/Orleans/Orleans.csproj

```diff
@@ -71,27 +71,18 @@
     <WarningsAsErrors>4014</WarningsAsErrors>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="Microsoft.Data.Edm">
```

Comment:
It is not used by any code in Orleans.dll, and build doesn't break if it is nixed.

Having said that though, on second look i will leave it in for consistency with the other projects that pull in WindowsAzure.Storage package, which lists Data.Edm as a dependency even though that list actually seems overly broad.